src/handlers/favorites/addFavoriteHandler.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+import { Settings } from "../../Settings";
+import * as vscode from "vscode";
+
+export async function addFavoriteHandler() {
+
+    const alias = await vscode.window.showInputBox({
```

Comment:
Users care the command more then alias, and actually they even don't need an alias, maybe.

So I would prefer to prompt for `command` first, and then `alias`, and use the command as default value when asking to input `alias`. So those who don't care about `alias` can press enter all the way to go.